docs/extend/index.md

```diff
@@ -97,7 +97,7 @@ started without error.
     ```bash
     $ docker volume ls
 
-    DRIVER              NAME
+    DRIVER              VOLUME NAME
     vieux/sshfs         sshvolume
```

Comment:
Perhaps in https://github.com/docker/docker/commit/a488ad1a090eb0b6069f79cd045badaaa8ec1bda